src/mame/drivers/pgm2.cpp

```diff
@@ -1,4 +1,4 @@
-// license:BSD-3-Clause
+﻿// license:BSD-3-Clause
```

Comment:
Why did this line change?  Have you inadvertently added a zero-width no-break space?

Reply:
it just mistake, i fixed now.